components/02-molecules/registration/SetTextRecordsAddressesComponent.tsx

```diff
@@ -67,6 +68,40 @@ export const SetTextRecordsAddressesComponent = ({
     }
   }, [authedAddress]);
 
+  useEffect(() => {
+    if (
+      nameRegistrationData?.domainAddresses &&
+      Object.keys(nameRegistrationData?.domainAddresses).length > 0
+    ) {
+      const addresses = Object.keys(
+        nameRegistrationData.domainAddresses
+      ).reduce((acc, key) => {
+        return {
+          ...acc,
+          [key]: {
+            address: nameRegistrationData.domainAddresses[key],
+            isValid: true,
+          },
+        };
+      }, {});
+
+      setAddresses(addresses);
+    }
+  }, []);
+
+  const saveDomainAddressesInLocalStorage = (
+    domainAddresses: Record<string, string>
+  ) => {
+    if (authedAddress && nameRegistrationData.name) {
```

Comment:
```suggestion
    if (address && nameRegistrationData.name) {
```

Reply:
Since 'address' is the default name of the wagmi address from useAccount(), we can keep it as address and specify better the other addresses occurrences, in my opinion